code/game/objects/items.dm

```diff
@@ -79,6 +79,10 @@
 
 /obj/item/New()
 	..()
+	if(force < 5) // Arbitrarily chosen
```

Comment:
Make it check if it already has a defined hit sound, otherwise this will override every hitsound out there